### ISSUE TITLE
questions about Attribute Recognition

### ISSUE THREAD

**Issue by Bepbebop** [xushilin1/FashionFormer#18]
Hi, sorry to bother you. Your work is really magnificiant. But here I get some questions about the attribute recognition part. In this project, is there any code that can output the attribute recognition of each image? Or if there isn't, could you please teach me how to get the output of attribute recognition? I'm stuck in achieving the cloth attribute recognition, running out of time,and your project is the only channce for me now. So this is really important to me, and if this is solved, I can pay you back with money if you want. Thank you!!!

**Comment by kampelmuehler** [comment]
Hi,
the results returned from `inference_detector()` include the attribute prediction logits (it's the 3rd entry in the results tuple). The translation of ids to human readable labels can be found in [this json](https://s3.amazonaws.com/ifashionist-dataset/annotations/instances_attributes_val2020.json).

Talking about this line here:
https://github.com/xushilin1/FashionFormer/blob/0752737bb1e071ebd03315a1c5e8a62ab2573620/demo/image_demo.py#L25
